### PR TITLE
Create Post Screen Added

### PR DESCRIPTION
**Pull request by Aditya13s**

### CODE REVIEW

**Comment by tusharojha** [comment]
@Aditya13s Please attach a screenshot of the app.

**Review by sidsharma2002 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/doubtless/doubtless/screens/post/CreatePostFragment.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        binding.postButton.setOnClickListener {
+            Toast.makeText(context, binding.postContent.text, Toast.LENGTH_SHORT).show()
+            createPost(binding.postContent.text.toString())
+        }
+    }
+
+    private fun createPost(text: String) {
+        val post = hashMapOf(
+            "Question" to text,
+            "Time" to Date(),
```
[the remark this answers is not in the document]
Comment (Aditya13s) [reply app/src/main/java/com/doubtless/doubtless/screens/post/CreatePostFragment.kt:42]: Yes Date().toString can be converted back to Date object using Date.parse(string)

File: app/src/main/res/layout/fragment_view_post.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
[the remark this answers is not in the document]
Comment (Aditya13s) [reply app/src/main/res/layout/fragment_view_post.xml:2]: Replaced Frame Layout with Constraint Layout

**Comment by Aditya13s** [comment]
Not Needed

**Pull request closed without merging**